target/linux/ath79/generic/base-files/lib/functions/mikrotik-caldata.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
This file is sourced only and has no executable bit set. So this should be removed.
I've already done that in my staging area.